build.gradle

```diff
@@ -55,15 +54,26 @@ tasks.withType(JavaCompile) {
 }
 
 // declare where to find the dependencies of your project
-repositories {
-    ['TBD'].each { repo ->
+allprojects {
+    repositories {
         maven {
+            url = uri("https://maven.pkg.github.com/Alvearie/hri-api-spec")
             credentials {
-                username = findProperty('user') ?: System.getenv('user')
-                password = findProperty('password') ?: System.getenv('password')
+                username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
+                password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
             }
+        }
+        mavenCentral()
+        mavenLocal()
+    }
+}
 
-            url "TBD/$repo/"
+repositories {
+    maven {
+        url = uri("https://maven.pkg.github.com/Alvearie/hri-api-spec")
+        credentials {
+            username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
+            password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
```

Comment:
It looks like we now have 2 different `repositories` sections in this file, one abive at line 58 and this one?  

Do we need both? Is this one redundant?

Reply:
One of the sections I have found to be unnecessary, I have removed that section.

Reply:
The other section is actually needed for publishing, it has been added back